ql/pricingengines/blackformula.hpp

```diff
@@ -372,6 +372,18 @@ namespace QuantLib {
                                          Real bachelierPrice,
                                          Real discount = 1.0);
 
+    /*! Exact Bachelier implied volatility
+
+        It is calculated using the analytic implied volatility formula
+        of Jaeckel (2017), "Implied Normal Volatility"
+    */
+    Real bachelierBlackFormulaImpliedVolExact(Option::Type optionType,
```

Comment:
Do you see any issues in having this replace the formula above, instead of keeping both?  Are there any cases in which we'd want the old one?

Reply:
No, I guess the default should be the Jaeckel one. We can keep the other one with a different name maybe?

Reply:
I made the new function the default and kept the other one under a separate name.